### ISSUE TITLE
 status code 500 (Selenium::WebDriver::Error::ServerError)

### ISSUE THREAD

**Issue by sam3sa** [SeleniumHQ/selenium#4211]
OS: MAC

Selenium Version: 3.4.3

Browser: Chromedriver

Browser Version: Version 59.0.3071.104 (Official Build) (64-bit)

Expected Behavior - Chrome launch successfully

Actual Behavior - status code 500 (Selenium::WebDriver::Error::ServerError)

Steps to reproduce -
using the below configuration to connect to a remote browser.
```
      ENV['HTTP_PROXY'] = ENV['http_proxy'] = nil
        # Selenium Grid Configurations
        url = 'http://grid_end_point/wd/hub'
        caps = Selenium::WebDriver::Remote::Capabilities.chrome
        Watir::Browser.new(
          :remote,
          :url => url,
          :desired_capabilities => caps
        )
```
**Note** This use to work with the previous version of ruby 3.4

**Comment by lmtierney** [comment]
Provide a way to reproduce with Selenium and not Watir please, otherwise I'm stabbing in thin air

**Comment by lmtierney** [comment]
Please provide [concise reproducible test case](http://sscce.org/) so that we can act on this issue. Once this is done the issue will be re-opened. See [CONTRIBUTING.md](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#issue-contributions)

**Comment by lmtierney** [comment]
Looks like a duplicate of #4195
